dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/SpringBindingTest.java

```diff
@@ -135,6 +135,14 @@ void shouldReturnABadRequestWhenInvalidValueForBooleanInBindingObjectIsPassedAsP
                         "Value 'INVALID' is not valid for parameter booleanValue. Invalid boolean value [INVALID]")));
   }
 
+  @Test
+  void shouldReturnADefaultValueWhenDefaultParameterIsNotPassed() throws Exception {
```

Comment:
could we also add a test to verify that the default value is not used if the parameter has a value?